doc/getting-started/getting-started.rst

```diff
@@ -657,10 +656,11 @@ Build ACRN
       the target system.
       Replace ``10.0.0.200`` with the target system's IP address you found earlier::
 
-         scp ~/acrn-work/acrn-hypervisor/build/acrn-my_board-MyConfiguration*.deb \
+         sudo scp ~/acrn-work/acrn*.deb \
+             ~/acrn-work/grub*.deb \
              ~/acrn-work/*acrn-service-vm*.deb \
              ~/acrn-work/MyConfiguration/launch_user_vm_id1.sh \
-             ~/acrn-work/acpica-unix-20210105/generate/unix/bin/iasl \
+             /usr/bin/iasl \
```

Comment:
Just checking: we have verified that the version installed from the Ubuntu repo is usable (since this is the version installed by the `acpica-tools` package)?

Reply:
Have verified the version 20200925 installed by the acpica-tools package workable including building ACRN packages, launching guests, generate ACPI tables compiler/disassembler.